### PR TITLE
Clean/docs structure

### PR DESCRIPTION
**Pull request by prdlk**
Description of PR...

## Changes

* Shifted Blockchain docs to root folder
* Moved vue wallet into web folder
* Changed config.yaml to resolve updated paths
* Updated Taskfile to follow colon notation
* Added Goreleaser CI/CD to build cli and provide releases
* Added Taskfile to web projects
* Moved Host into internal, our packaged highway and motor leverage host and does not need to be exposed

### `pkg/channel`

Moved all NewChannel logic into `internal/blockchain/x/channel/types` for standard implementation and removing duplication

 Mention [stepsize] in a comment if you'd like to report some technical debt. See examples here.

### CODE REVIEW

File: Taskfile.yaml [hunk 1]
```diff
@@ -1,36 +1,76 @@
...
+  build:swagger:
+    cmds:
+      - echo "Building swagger api docs for highway"
+      - swag i -g pkg/highway/server.go --output docs/highway
+      - echo "Building Open api docs for Blockchain"
+      - starport generate openapi
+      - starport generate proto-go
+      - go mod tidy
+
+  chain:serve:
```
Comment (mcjcloud) [inline Taskfile.yaml]: nit: You have `build:chain` but `chain:serve`. I think these should be made consistent

File: web/wallet/Taskfile.yaml [hunk 2]
```diff
@@ -19,4 +21,5 @@ tasks:
 
   serve:
     cmds:
-      - npm run dev
+      - npm install
```
Comment (ntindle) [inline web/wallet/Taskfile.yaml]: This command has side-effects. I would not recommend doing this as part of other steps without being clear

File: Taskfile.yaml (hunk 1 above)
Comment (prdlk) [reply Taskfile.yaml]: okay serve:chain

File: web/wallet/Taskfile.yaml (hunk 2 above)
Comment (prdlk) [reply web/wallet/Taskfile.yaml]: Okay removed im not a web dev XD

**Review by mcjcloud — APPROVED** [review]
lgtm

**Pull request merged**